tests/Unit/SDK/SystemClockTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Unit\SDK;
+
+use OpenTelemetry\SDK\SystemClock;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\SDK\SystemClock
+ */
+class SystemClockTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState false
+     */
+    public function test_get_instance(): void
+    {
+        $clock = SystemClock::getInstance();
+        $this->assertSame($clock, SystemClock::getInstance());
```

Comment:
`assertEquals`

Reply:
this is testing the singleton functionality, so ensuring it's the same object that comes back a second time (have updated the test name to be more descriptive)

Reply:
Yap, sorry, I was aware of that. I did not mean to post that comment, must have been by accident.